bamboo/docker-compose-local.yml

```diff
@@ -24,7 +24,7 @@ services:
       - 8080:8080
       - 9200:9200
   localstack:
-    image: localstack/localstack:0.12.13
+    image: localstack/localstack:1.3.1
```

Comment:
This isn't an insignificant change (for localstack), however looking at the migration notes for the major version change and/or the changelog (it's voluminous) I don't see any obvious pitfalls that would cause false passes in the units/incompats/etc.  

I'm going to pull it in and make sure it runs correctly on an intel mac and/or CI.

Reply:
@Jkovarik Have you been able to test this? Having this merged will make it easier for me to work on getting the ES tests working.